cloud-controller-manager/do/lb_annotations.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2020 DigitalOcean
```

Comment:
Could use the opportunity to update to 2022.

Reply:
Would say 2023 :)

Reply:
Haha yeah good catch. I'm still stuck in the past it seems. 😀